python/paddle/v2/model.py

```diff
@@ -33,37 +30,38 @@ def mkdir_p(path):
             raise
 
 
-def save_model(parameters, path):
+def save_model(parameters, folder, name="model.tar"):
     need_request = "KUBERNETES_SERVICE_HOST" in os.environ.keys()
 
     if need_request:
         # TODO(helin): figure out how MPI trains, since MPI only save
         # model when trainer_id == "0", we can consolidate the logic
         # here.
-
-        # TODO(helin): change this environment variable name from
-        # MASTER_IP to ETCD_IP
-        etcd_name = "MASTER_IP"
-        if etcd_name not in os.environ.keys():
-            raise Exception('not find ' + etcd_name +
+        trainer_key = "PADDLE_INIT_TRAINER_ID"
+        if trainer_key not in os.environ.keys():
+            raise Exception('not find ' + trainer_key +
                             ' in environment variable.')
-
-        etcd_ip = os.environ.get(etcd_name)
-        client = master.client("http://" + etcd_ip + ":2379", 5, 0)
-        r = client.request_save_model(trainer_id, 5000)
-        if r == 0:
-            # do not need to save
-            return
-        elif r < 0:
-            # error
-            return
+        trainer_id = os.environ.get(trainer_key)
+        etcd_key = "ETCD_IP"
```

Comment:
I think we set it here: https://github.com/PaddlePaddle/cloud/blob/cafa1329209b0682ddc17db074b1042b1cb97f45/docker/paddle_k8s#L64